2010-03-12  Michael Tautschnig  <mt@debian.org>

	* setup-storage/Commands.pm: Work around missing ext4 support in parted
		(thanks Stefan Goetz for the patch). (closes: #572120)
Index: trunk/lib/setup-storage/Commands.pm
===================================================================
--- trunk.orig/lib/setup-storage/Commands.pm
+++ trunk/lib/setup-storage/Commands.pm	
@@ -891,6 +891,7 @@
     $fs = "linux-swap" if ($fs eq "swap");
     $fs = "fat32" if ($fs eq "vfat");
     $fs = "fat16" if ($fs eq "msdos");
+    $fs = "ext3" if ($fs eq "ext4");
     $fs = $FAI::current_config{$disk}{partitions}{$mapped_id}{filesystem}
       if ($part->{size}->{preserve} || $part->{size}->{resize});
     $fs = "" if ($fs eq "-");
